src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Truck truck = new Truck();
+        Excavator excavator = new Excavator();
+        Bulldozer bulldozer = new Bulldozer();
 
+        Machine[] machines = new Machine[]{excavator,bulldozer,truck};
+
+        for (Machine machine : machines) {
+            machine.doWork();
```

Comment:
The message indicating what type of Machine is working is missing. It's important to have informative messages to know which Machine is currently doing work.